lib/qa/authorities/getty.rb

```diff
@@ -3,14 +3,23 @@
 module Qa::Authorities
   module Getty
     require 'qa/authorities/getty/aat'
+    require 'qa/authorities/getty/tgn'
+    require 'qa/authorities/getty/ulan'
     extend AuthorityWithSubAuthority
 
     def self.subauthorities
-      [ "aat" ]
+      [ "aat" , "tgn", "ulan" ]
     end
 
-    def self.subauthority_class(_)
```

Comment:
Please don't remove this. Use this to return the class rather than overriding subauthority_for to return an instance.